browser/help/faq/mitochondrial-dna/mtdnaHaplogroupPerNuclearAncestryPopulationData.ts

```diff
@@ -0,0 +1,247 @@
+export const haplogroups = [
+  'A',
+  'B',
+  'C',
+  'D',
+  'E',
+  'F',
+  'G',
+  'H',
+  'HV',
+  'I',
+  'J',
+  'K',
+  'L0',
+  'L1',
+  'L2',
+  'L3',
+  'L4',
+  'L5',
+  'M',
+  'N',
+  'P',
+  'R',
+  'T',
+  'U',
+  'V',
+  'W',
+  'X',
+  'Y',
+  'Z',
+] as const
+
+export type Haplogroup = typeof haplogroups[number]
```

Comment:
aha, this was what caused you to make that post in the SoftEng slack channel about this topic. Interesting.

Reply:
Yeah, it bothered me that I apparently had to repeat myself at the type level and at the value level, so I dug a little and found out this little gem. Typescript's type system is a big and sophisticated one so don't worry about trying to take it all in at once.